servers/quarkus-server/src/testFixtures/java/org/projectnessie/server/catalog/ViewCatalogTests.java

```diff
@@ -15,9 +15,15 @@
  */
 package org.projectnessie.server.catalog;
 
+// Marker for Nessie LICENSE file - keep it
+// CODE_COPIED_TO_NESSIE
```

Comment:
Don't we need this in `CatalogTests` as well?

Reply:
It's there.